docker-compose.yml

```diff
@@ -35,3 +35,14 @@ services:
       JAVA_OPTS: "-Dserver.port=8090 -Dchaos.strategy=INTERNAL_SERVER_ERROR -Ddestination.hostProtocolAndPort=http://scruid_druid:8082"
     ports:
       - "8086:8090"
+
+  slow_response:
+    build:
+      context: docker/openresty
+    container_name: slow_response
+    depends_on:
+      - scruid_druid
```

Comment:
why does it depends on scruid_druid ?

Reply:
Because I copied that from the other two without giving it too much thought. I suppose that at this stage it doesn't; nor does it require the container link. 

However, the embedded NGINX could at some point be modified to proxy the Druid server behind it (for example, to add basic authentication without juggling Druid configuration files and restarting the Druid processes). The prototype code that @bjgbeelen and @anskarl wrote regarding #66 adds authentication that way; we could re-use this NGINX container and avoid adding another one.

Reply:
ok, fair enough